src/screens/Addresses/AddressesScreen.tsx

```diff
@@ -16,28 +16,26 @@ You should have received a copy of the GNU Lesser General Public License
 along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
-import { StackScreenProps } from '@react-navigation/stack'
-import { colord } from 'colord'
-import { ListIcon, PlusIcon, Upload } from 'lucide-react-native'
 import { usePostHog } from 'posthog-react-native'
 import { useEffect, useState } from 'react'
-import { RefreshControl, View } from 'react-native'
-import { useModalize } from 'react-native-modalize'
+import { View } from 'react-native'
+import { FlatList } from 'react-native-gesture-handler'
```

Comment:
Is there a specific reason why this is imported from `react-native-gesture-handler` and not `react-native`?

Reply:
Mhh interesting! I didn't notice it. https://docs.swmansion.com/react-native-gesture-handler/docs/1.10.3/about-handlers/#using-native-components

In this case, I don't think that we leverage the `react-native-gesture-handler` gesture system, so this isn't necessary.